northstar/src/runtime/mod.rs

```diff
@@ -60,10 +60,33 @@ pub(self) type RepositoryId = String;
 pub use api::container::*;
 pub(self) use repository::Repository;
 pub(self) use state::MountedContainer;
-
 pub(self) type ExitCode = i32;
 pub(self) type Pid = u32;
 
+#[async_trait]
+pub(self) trait Process {
```

Comment:
why do you use `pub(self)`? isn't it the same like using nothing?
If it is just used in the current module at least

Reply:
Correct.